numba/numpy_support.py

```diff
@@ -97,6 +116,16 @@ def from_dtype(dtype):
     else:
         return from_struct_dtype(dtype)
 
+def to_dtype(numba_type):
```

Comment:
Ah, good question - the problem with `as_dtype` was that it returns a `numpy.dtype` instance, which cannot be called to construct an instance of the type (as opposed to something like `np.int32`, etc., which can be), which I was doing to get the itemsize. However, I have found that I don't need an instance to get the itemsize, so I've removed `to_dtype`.